tests/blackjack_tests.cpp

```diff
@@ -1123,6 +1123,26 @@ BOOST_FIXTURE_TEST_CASE(max_payout_split, blackjack_tester) {
     check_player_win(STRSYM("100000.0000"));
 }
 
+BOOST_FIXTURE_TEST_CASE(cant_deposit_after_stand, blackjack_tester) try {
+    const auto ses_id = new_game_session(game_name, player_name, casino_id, STRSYM("100.0000"));
+    bet(ses_id, STRSYM("100.0000"));
+
+    // Tc is a hole card
+    push_cards(ses_id, {"Kd", "Ts", "Td"});
+    signidice(game_name, ses_id);
```

Comment:
after stand the game finishes, session is deleted
so, technically, after transfer in this scenario you shouldn't get an error
See this
https://github.com/DaoCasino/game-contract-sdk/blob/master/sdk/include/game-contract-sdk/game_base.hpp#L335

interesting cases to check are when you deposit when the game doesn't expect it, i.e
- deposit after hit
- deposit after split

In the above cases we should get an error since it violates the rules

Reply:
added test with deposit after hit, after split we can double down so it's legal deposit